packages/block-editor/src/components/block-bindings-toolbar-indicator/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { ToolbarItem, ToolbarGroup, Icon } from '@wordpress/components';
+import { connection } from '@wordpress/icons';
+import { _x } from '@wordpress/i18n';
+
+export default function BlockBindingsToolbarIndicator() {
+	return (
+		<ToolbarGroup>
+			<ToolbarItem
+				as={ 'div' }
+				aria-label={ _x( 'Connected', 'block toolbar button label' ) }
+				className="block-editor-block-bindings-indicator block-toolbar-block-bindings-indicator"
```

Comment:
I checked the guidelines for [CSS naming for React components](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md#naming) so it would better fit to have:

```suggestion
				className="block-editor-block-bindings-toolbar-indicator"
```

I don't know how much this pattern is being followed in practice these days. We definitely need only one class name here.

Reply:
I'm being pedantic here but reading the guideline, shouldn't it be `block-editor-block__bindings-toolbar-indicator` ?

> Any descendent of the component's root element must append a dash-delimited descriptor, separated from the base by two consecutive underscores __.
>
> Root element: package-directory
> Child elements: package-directory__descriptor-foo-bar

Reply:
In the grand scheme of things it's not super important but I've updated it in 3f9fa3bc2622ef8af610816e3c75e1a387c159c0. 

We add some of the styles to the `BlockList` component [here](https://github.com/WordPress/gutenberg/blob/3f9fa3bc2622ef8af610816e3c75e1a387c159c0/packages/block-editor/src/components/list-view/block-select-button.js#L291-L292) as well. So having two classes makes sense here: 
-  `block-editor__block-bindings-indicator` is used in both
- `block-toolbar__block-bindings-indicator` is used only in the toolbar.

I'm not 100% sure if we should put both styles inside of https://github.com/WordPress/gutenberg/blob/3f9fa3bc2622ef8af610816e3c75e1a387c159c0/packages/block-editor/src/components/block-bindings-toolbar-indicator/style.scss but I don't see a better place for shared styles.

Reply:
> Root element: package-directory

Package is `@wordpress/block-editor`.
Directory is `block-bindings-toolbar-indicator`.

The `__` part comes into play when you put additional classed as child elements or when you have more components in different files than `index.js` in the same folder.

Reply:
Uh, right! I misunderstood something there. I'll adjust it now.

Reply:
We might be able to get rid of the second class — checking now

Reply:
Never mind — your recent changes look better than what I had in mind.

Reply:
> I'm being pedantic here but reading the guideline, shouldn't it be `block-editor-block__bindings-toolbar-indicator` ?

Yeah, I meant to write `block-editor__block-bindings-toolbar-indicator` there, adding to the confusion.

Instead, now I used `block-editor-list-view-block-select-button__bindings` and `block-editor-block-bindings-toolbar-indicator` which should be in line with the convention in GB.